Makefile

```
@@ -178,7 +178,7 @@ ci-lint: golangci-lint
 
 .PHONY: verify
 verify: gomod-verify vet ci-lint fmt-verify toc-verify manifests generate update-helm
-	git --no-pager diff --exit-code config/components apis charts/kueue/templates/crd client-go
+	git --no-pager diff --exit-code config/components apis charts/kueue/templates client-go
```

```suggestion
	git --no-pager diff --exit-code config/components apis charts/kueue/templates/crd charts/kueue/templates/rbac client-go
```

Should we verify only `crd` and `rbac` since [`update-helm.sh](https://github.com/kubernetes-sigs/kueue/blob/3d1dc9a4fb78c52bdb8ca53d8c94139a6f241304/hack/update-helm.sh)` syncs only those folders?

---

In my opinion no.  The current way it's easier to follow, and if in the future other templates are auto updated are automatically updated we will not miss to update the check.

---

I don't have a strong objection. I was just concerned that it would get the error in this script even when we changed the helm templates, which requires manual changes.

---

> if in the future other templates are auto updated are automatically updated we will not miss to update the check.

Anyway, It makes sense on this.